ep006/README.md

```diff
@@ -0,0 +1,40 @@
+# [python type(x).__name__ vs x.__class__.__name__ (intermediate)](https://youtu.be/6rAIttnm3Fs)
+
+Very short one, I explain why I prefer type(x).__name__ vs x.__class__.__name__.  thanks to @alikus for the question!
+
+## Interactive examples
+
+### Python
+
+```python
+# python3.8
+Location()
+class C:
+    @property
+    def __class__(self):
+        print('return custom type')
+        return int
+
+C().__class__
+C().__class__.__name__
```

Comment:
I think that you meant to show the `.__class__.__name__` attribute, so I added it as well.